src/index.js

```diff
@@ -7,19 +7,19 @@ import App from './App'
 import Config from './config/config'
 import bugsnag from '@bugsnag/js'
 import bugsnagReact from '@bugsnag/plugin-react'
+import fontMaterialIcons from 'react-native-vector-icons/Fonts/MaterialIcons.ttf'
```

Comment:
dont we need a special file for web and native?

Reply:
No, Native ones get loaded by the native code